<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="pbxcore"
           tests="9"
           failures="2"
           errors="0"
           skipped="0"
           time="308.799">
  <properties />
            <testcase name="C_calldefault1_sip" time="27.639" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback">
                <failure type="java.lang.AssertionError"
                  message="预期3001会响铃 【预期：2 实际：1】"
              >
<![CDATA[java.lang.AssertionError: 预期3001会响铃 【预期：2 实际：1】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:429)
  com.yeastar.swebtest.driver.SwebDriver.ysAssertWithHangup(SwebDriver.java:912)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback.C_calldefault1_sip(Callback.java:90)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
            <testcase name="E_callbackThroughSPS" time="27.635" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback">
                <failure type="java.lang.AssertionError"
                  message="预期辅助2的2000分机响铃 【预期：2 实际：4】"
              >
<![CDATA[java.lang.AssertionError: 预期辅助2的2000分机响铃 【预期：2 实际：4】
  com.yeastar.swebtest.tools.ysassert.YsAssert.fail(YsAssert.java:1092)
  com.yeastar.swebtest.tools.ysassert.YsAssert.failNotEquals(YsAssert.java:583)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEqualsImpl(YsAssert.java:146)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:120)
  com.yeastar.swebtest.tools.ysassert.YsAssert.assertEquals(YsAssert.java:429)
  com.yeastar.swebtest.driver.SwebDriver.ysAssertWithHangup(SwebDriver.java:912)
  com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback.E_callbackThroughSPS(Callback.java:138)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
  org.testng.TestRunner.privateRun(TestRunner.java:774)
  org.testng.TestRunner.run(TestRunner.java:624)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
  org.testng.SuiteRunner.run(SuiteRunner.java:261)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.run(TestNG.java:1048)
  org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:293)
  org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
  org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:91)
  org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:200)
  org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:153)
  org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="C_calldefault2_sps" time="41.098" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback" />
                <testcase name="A1_add_callback" time="59.500" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback" />
                <testcase name="D_editInbound1" time="31.277" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback" />
                <testcase name="F_delete" time="47.973" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback" />
                <testcase name="A0_Register" time="10.104" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback" />
                <testcase name="G_Recovery" time="30.924" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback" />
                <testcase name="B_editInbound1" time="32.649" classname="com.yeastar.swebtest.testcase.RegressionCase.pbxcase.Callback" />
      
</testsuite>
